.travis.yml

```diff
@@ -20,13 +20,15 @@ env:
 before_install:
   - sudo apt-get install shellcheck
   - sudo pip install yamllint
+  - git clone https://github.com/IBM/ci-repo
```

Comment:
Last nit about this - using 'repo' in the name of a repo isn't very descriptive.  In keeping with convention set up by the watson team, I would suggest a name along the lines of 'pattern-ci'